neurox/interpretation/probeless.py

```diff
@@ -0,0 +1,118 @@
+"""Module for Probeless method
+This module extracts neuron ranking for an attribute or for the entire property set .
+.. see also::
+        `Antverg, Omer and Belinkov, Yonatan "On The Pitfalls Of Analyzing Idividual Neurons in Language Models." In Proceedings of the 10th International Conference on Learning Representations (ICLR).
+"""
+import numpy as np
+import torch
+import itertools
+import sys
+import torch.nn as nn
+from torch.autograd import Variable
+
+from . import metrics
+from . import utils
+from . import ablation
+
+
+def get_mean_vectors(X_t, Y_t):
+    
+    embeddings_by_labels = {}
+    average_embeddings_by_label = {}
+    rankings_per_label = {}
+
+    for words in range(Y_t.size):
```

Comment:
Append `_idx` to index variables so its easy to understand the code that follows; i.e. `words` -> `word_idx`